models/balancer/arbitrum/balancer_v2_arbitrum_liquidity.sql

```diff
@@ -67,14 +67,14 @@ WITH pool_labels AS (
             day,
             pool_id,
             token,
-            SUM(COALESCE(delta, 0)) AS delta
+            SUM(COALESCE(delta, CAST(0 as int256))) AS delta
         FROM
             (
                 SELECT
                     date_trunc('day', evt_block_time) AS day,
                     poolId AS pool_id,
                     tokenIn AS token,
-                    amountIn AS delta
+                    CAST(amountIn as int256) AS delta
```

Comment:
good example here of `CAST()` being preferred
(note, dunesql engine preference for performance)